packages/headless/src/controllers/core/facets/facet/headless-core-facet.ts

```diff
@@ -209,6 +209,10 @@ export interface CoreFacetState {
 
   /** Whether the facet is enabled and its values are used to filter search results. */
   enabled: boolean;
+  /**
+   * The name to display if this field is used by the Facet Generator in your interface.
```

Comment:
```suggestion
   * The name to display if this field is used by the Facet Generator in your interface. See [Change Facet Generator options](https://docs.coveo.com/en/n9sd0159#change-facet-generator-options).
```

Cause I don't think it's gonna be super obvious to most people where this label would be coming from and how they can configure it 😅